__tests__/posts.ts

```diff
@@ -3473,6 +3471,94 @@ describe('query sourcePostModeration', () => {
   });
 });
 
+describe('mutation createSourcePostModeration', () => {
```

Comment:
I think we should add some tests for user (member/mod/admin) trying to post in squad directly. 
This is an important point for this PR imo 👀